tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/compiler/CompilerTest.java

```diff
@@ -63,6 +74,25 @@
  */
 public class CompilerTest extends AbstractXtendTestCase {
 	
+	@Test public void testBug480818() throws Exception {
```

Comment:
There is `org.eclipse.xtend.core.tests.parsing.PartialParserTest` which has utility methods for partial parsing assertions.